rts/Sim/MoveTypes/HoverAirMoveType.cpp

```diff
@@ -410,7 +410,7 @@ void CHoverAirMoveType::UpdateHovering()
 
 	// randomly drift (but not too far from goal-position)
 	wantedSpeed = (randomWind * math::fabs(owner->unitDef->dlHoverFactor) * 0.5f);
-	wantedSpeed += (smoothstep(0.0f, 20.0f * 20.0f, (goalPos - owner->pos)) * (goalPos - owner->pos));
+	wantedSpeed += (goalPos - owner->pos) * 0.1f;
```

Comment:
While smoothstep was indeed wrong here because it doesn't take into account where the movement started, this solution is bad as well and can easily cause overshoot.